### PR TITLE
Add user game after login

### PR DESCRIPTION
**Pull request by arimet**
- [x] Fetch User' games
- [x] Render in GameSelector view
- [x] Add Game Preview (if possible)
image

### CODE REVIEW

**Comment by alexisjanvier** [comment]
@arimet need rebase

File: src/GameSelector/index.tsx [hunk 1]
```diff
@@ -1,70 +1,146 @@
...
   };
 
-  const handleChangeText = (value: string) => {
-    setData({ ...formData, gameId: +value });
+  useEffect(() => {
+    getUserGames();
+  }, []);
+
+  const UserGamesList = () => {
+    return (
```
Comment (alexisjanvier) [inline src/GameSelector/index.tsx:72]: What do you display in case there is an `errorUserGames` ?

File: src/GameState/index.tsx [hunk 2]
```diff
@@ -104,7 +100,7 @@ export default function GameState({ navigation, route }: Props) {
         >
           <Spinner size="lg" />
         </View>
-      ) : state.error ? (
+      ) : !state.error ? (
```
Comment (alexisjanvier) [inline src/GameState/index.tsx:114]: oh yes ...

File: src/GameState/Board.tsx [hunk 3]
```diff
@@ -1,32 +1,34 @@
...
               justifyContent="center"
               key={cellIndex}
               bg="#948e8b"
               m={0}
               p={0}
             >
-              <Marble value={cell}/>
+              <Marble
+                value={cell}
+                size={cell === 0 ? marbleEmptySize : marbleColorSize}
```
Comment (alexisjanvier) [inline src/GameState/Board.tsx:26]: IMO this logic should be handled on the `Marble` component side

**Review by alexisjanvier — CHANGES_REQUESTED** [review]

File: src/GameSelector/index.tsx (hunk 1 above)
Comment (arimet) [reply src/GameSelector/index.tsx:72]: I just added a toast to warn if the fetch was not successful. 
And also a message if the player has no game in progress

File: src/GameState/Board.tsx (hunk 3 above)
Comment (arimet) [reply src/GameState/Board.tsx:26]: Do you think? 
Shouldn't the marble component just display a marble without caring about its context?

File: src/GameSelector/index.tsx (hunk 1 above)
Comment (alexisjanvier) [reply src/GameSelector/index.tsx:72]: ok

File: src/GameState/Board.tsx (hunk 3 above)
Comment (alexisjanvier) [reply src/GameState/Board.tsx:26]: It's not wrong. I don't really like the ... *mamaille* on lines 7 to 9. But okay.

**Review by alexisjanvier — APPROVED** [review]

**Pull request merged**